distribution/changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.0.7 (in development)
 ------------------------------------------------------------------------
 - Feature: [#5110] The tile inspector can now be used in multiplayer.
+- Feature: [#5254] Scenario options are now syncronised over multiplayer.
```

Comment:
Due to many issues, I am no longer working on this project at the time. If you wish to spell it correctly, you have write access, so you can do so yourself, otherwise it will have to wait a few weeks.